src/ovirtapi/index.js

```diff
@@ -435,6 +435,13 @@ const OvirtApi = {
     })
   },
 
+  getSpiceUsbAutoShare (): Promise<Object> {
```

Comment:
`getOption` (and `getOptionWithoutDefault`) already handle this case in a more general way.  No need to have a function call for each setting.